src/onefetch/cli.rs

```diff
@@ -173,6 +183,17 @@ impl Cli {
             None
         };
 
+        let image_colors = if let Some(value) = matches.value_of("image-colors") {
+            let colors = usize::from_str(value).unwrap();
```

Comment:
If `Arg::possible_values` is not used, then it could be possible for a user to enter a non-number value, so unwrapping here may not be safe.